distil/primitives/audio_transfer.py

```diff
@@ -86,6 +86,10 @@ class AudioTransferPrimitive(unsupervised_learning.UnsupervisedLearnerPrimitiveB
                     'package_uri': 'git+https://github.com/uncharted-distil/distil-primitives.git@{git_commit}#egg=distil-primitives'.format(
                         git_commit=utils.current_git_commit(os.path.dirname(__file__)),
                     )
+                }, {
+                    'type': metadata_base.PrimitiveInstallationType.PIP,
```

Comment:
Sorry to bother you here more, but:
* The order is important, so this should be before you package.
* You should use `0.29.14` version. This is the one which will be fixed program-wide for next release.

Reply:
No worries. Thank you for your help. The order and version have been updated.